src/pretix/control/templates/pretixcontrol/event/mail.html

```diff
@@ -13,106 +14,13 @@
-                        <div class="panel-body">
-                            {% bootstrap_field form.mail_text_order_changed layout="horizontal" %}
-                        </div>
-                    </div>
-                </div>
-                <div class="panel panel-default">
-                    <div class="panel-heading">
-                        <h4 class="panel-title">
-                            <a data-toggle="collapse" href="#order_expirew">
-                                <strong>{% trans "Payment reminder" %}</strong>
-                            </a>
-                        </h4>
-                    </div>
-                    <div id="order_expirew" class="panel-collapse collapse">
-                        <div class="panel-body">
-                            {% bootstrap_field form.mail_days_order_expire_warning layout="horizontal" %}
-                            {% bootstrap_field form.mail_text_order_expire_warning layout="horizontal" %}
-                        </div>
-                    </div>
-                </div>
-                <div class="panel panel-default">
-                    <div class="panel-heading">
-                        <h4 class="panel-title">
-                            <a data-toggle="collapse" href="#waiting_list">
-                                <strong>{% trans "Waiting list notification" %}</strong>
-                            </a>
-                        </h4>
-                    </div>
-                    <div id="waiting_list" class="panel-collapse collapse">
-                        <div class="panel-body">
-                            {% bootstrap_field form.mail_text_waiting_list layout="horizontal" %}
-                        </div>
-                    </div>
-                </div>
+                {% include "pretixcontrol/event/mail_settings_fragment.html" with pid="order_placed" title="Placed order" items="mail_text_order_placed" %}
+                {% include "pretixcontrol/event/mail_settings_fragment.html" with pid="order_paid" title="Paid order" items="mail_text_order_paid" %}
+                {% include "pretixcontrol/event/mail_settings_fragment.html" with pid="order_free" title="Free order" items="mail_text_order_free" %}
+                {% include "pretixcontrol/event/mail_settings_fragment.html" with pid="resend_link" title="Resend link" items="mail_text_resend_link,mail_text_resend_all_links" %}
+                {% include "pretixcontrol/event/mail_settings_fragment.html" with pid="order_changed" title="Order changed" items="mail_text_order_changed" %}
+                {% include "pretixcontrol/event/mail_settings_fragment.html" with pid="order_expirew" title="Payment reminder" items="mail_days_order_expire_warning,mail_text_order_expire_warning" exclude="mail_days_order_expire_warning" %}
+                {% include "pretixcontrol/event/mail_settings_fragment.html" with pid="waiting_list" title="Waiting list notification" items="mail_text_waiting_list" %}
```

Comment:
You surprised me with the ``|getattr:``, I was wondering how you're going to do that but well, it seems to work and I don't have a cleaner solution right now ;)

However, we need to treat the title differently so the strings show up correctly in the translation, for example:
```
{% blocktrans asvar the_title %}Placed order{% endblocktrans %}
{% include "pretixcontrol/event/mail_settings_fragment.html" with pid="order_placed" title=the_title items="mail_text_order_placed" %}
```

Also, if you look at the ``resend_link`` part, there are two different text fields that are currently both labeled with ``Text``, we probably need to pass alternative labels here. Maybe you try to just use the ``label`` attribute of the form field? it should be filled correctly.